server/graphql/general/src/queries/generate_inbound_return_lines.rs

```diff
@@ -1,43 +1,75 @@
 use async_graphql::*;
-use chrono::NaiveDate;
+
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use graphql_types::types::GeneratedInboundReturnLineConnector;
+use service::auth::{Resource, ResourceAccessRequest};
+
+use service::invoice::inbound_return::{
+    ExistingLinesInput as ExistingLinesServiceInput,
+    GenerateInboundReturnLinesInput as ServiceInput,
+};
+
+#[derive(InputObject, Clone)]
+pub struct ExistingLinesInput {
+    pub item_id: String,
+    pub return_id: String,
+}
 
 #[derive(InputObject, Clone)]
-pub struct GenerateInboundReturnInput {
-    pub stock_line_ids: Vec<String>,
+pub struct GenerateInboundReturnLinesInput {
+    /// The ids of the outbound shipment lines to generate new return lines for
+    pub outbound_shipment_line_ids: Vec<String>,
```

Comment:
In the PR description you have: `or returning existing return lines for a specific itemId.`, that must be in the follow up pr right ?

Reply:
Scrap that, i didn't see the existing_line_input thing, thanks